sqlsynthgen/providers.py

```diff
@@ -20,7 +20,7 @@ def column_value(self, db_connection: Any, orm_class: Any, column_name: str) ->
         """Return a random value from the column specified."""
         query = select(orm_class).order_by(func.random()).limit(1)
         random_row = db_connection.execute(query).first()
-        return getattr(random_row, column_name)
+        return getattr(random_row, column_name, None)
```

Comment:
This might actually be better because we are happy to ignore empty tables but probably want to error if `column_name` isn't in the table.

```suggestion
        if random_row:
            return getattr(random_row, column_name)
```